### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by emmaindal**
## Hey, I just made a Pull Request!

Continuation of [642](https://github.com/spotify/backstage/pull/642)

Skärmavbild 2020-04-29 kl  23 42 50

**Darkmode:**

Skärmavbild 2020-04-30 kl  17 17 08

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/core/src/layout/ErrorPage/ErrorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    padding: theme.spacing(8),
+  },
+  title: {
+    paddingBottom: theme.spacing(5),
+  },
+  subtitle: {
+    color: theme.palette.grey[600],
```
Comment (stefanalund) [inline packages/core/src/layout/ErrorPage/ErrorPage.tsx:38]: How will this behave in dark mode?

Comment (emmaindal) [reply packages/core/src/layout/ErrorPage/ErrorPage.tsx]: Added picture above!

Comment (emmaindal) [reply packages/core/src/layout/ErrorPage/ErrorPage.tsx]: And also changed color from palette!

**Review by stefanalund — APPROVED** [review]
👏🏼 Happy first contribution Emma!

**Pull request merged**